### ISSUE TITLE
Culture is not supported. km_kh is invalid culture identifier.

### ISSUE THREAD

**Issue by Ferminatos** [fullstackhero/blazor-starter-kit#348]
Am using latest visual studio version but when I try to run the application an alert window appears saying "unable to connect to WarehouseManger.Server" what am I doing wrong I really couldn't figure it out.
I tried running in debugging mode and use IS server express instead but it couldn't work it also  gives a different kind of error saying "cultureInfo is not supported" on multiple lines of different files. Can you please help

1. Just trying to run the application for the first time

Expected to happen
The obvious, I expect to 

Desktop (please complete the following information):
 - OS: windows
 - Browser: chrome
 - Version [ ]

**Comment by jeremiedevos** [comment]
Did you follow the instructions mentioned at : https://codewithmukesh.com/blog/blazor-hero-quick-start-guide/
You need to have dotnet 5 and the WarehouseManager.Server as startup project.

**Comment by Ferminatos** [comment]
First of all thanks for your response. To answer your question yes I followed all the steps detailed on the post as well as on the YouTube video and I have the latest .Net 5 and visual studio(version 16.11) installed and I have checked if. Net 5 is installed correctly, it has and I also made WarehouseManger.server as my start up project. What it shows an error that says  "culturenotfoundexception: culture is not supported. (parameter 'name') km_kh is an invalid culture identifier. system.globalization.cultureinfo..ctor(string name, bool useuseroverride) culturenotfoundexception: culture is not supported. (parameter 'name') km_kh is an invalid culture identifier." I found out this kind of errors usually are associated with the language and region of your machine. Am running windows 8.1 on my laptop does that might be the case?
